### PR TITLE
Fix the flipgravity() internal command

### PR DESCRIPTION
**Pull request by InfoTeddy**
## Legal Stuff:

By submitting this pull request, I confirm that...

- [x] My changes may be used in a future commercial release of VVVVVV (for
  example, a 2.3 update on Steam for Windows/macOS/Linux)
- [x] I will be credited in a `CONTRIBUTORS` file for all of said releases, but
  will NOT be compensated for these changes

## Changes:

* **Fix the `flipgravity()` internal command**

  There are two main problems with `flipgravity()`:
  ​ ​ 1. It doesn't work for an already-flipped crewmate.
  ​ ​ 2. It doesn't work on the player.

  This PR addresses both of those issues.

### CODE REVIEW

**Comment by flibitijibibo** [comment]
This seems like something that would break compatibility - are we aware of anything that might depend on this behavior?

File: desktop_version/src/Script.cpp [hunk 1]
```diff
@@ -954,8 +950,23 @@ void scriptclass::run( KeyPoll& key, Graphics& dwgfx, Game& game, mapclass& map,
...
 				}
 
-				obj.entities[i].rule =7;
-				obj.entities[i].tile = 6;
+				if (words[1] == "player")
+				{
+					game.gravitycontrol = !game.gravitycontrol;
+				}
+				else
+				{
```
Comment (JayFoxRox) [inline desktop_version/src/Script.cpp]: For code readability the `getcrewman` if/else from above could be moved  into this scope.

**Comment by InfoTeddy** [comment]
@flibitijibibo I don't think anyone's deliberately inserting invalid `flipgravity(player)` commands into their level. I ran `grep -io 'flipgravity[(,)]player' more_levels/*.vvvvvv | wc -l`, and it returned 0 results. My `more_levels/` folder contains a huge archive of levels, and according to `ls more_levels | wc -l` it contains 1286 files.

As for using `flipgravity()` on an already-flipped crewmate, there might be one or two levels that have done this accidentally and have forgotten to take it out even though it doesn't work, but I don't know of any. Usually a level creator would realize that it doesn't work and find a workaround through using `gotoroom()`. But unlike `flipgravity(player)`, if `flipgravity()` is accidentally called on an already-flipped crewmate, the effect is purely visual and there's nothing I can think of that depends on crewmates' positions in custom levels (whereas if `flipgravity(player)` was called when it shouldn't be, that would be breaking gameplay).

File: desktop_version/src/Script.cpp (hunk 1 above)
Comment (InfoTeddy) [reply desktop_version/src/Script.cpp]: Fixed in 7173fce493453e903fd01133e58c950ef5128d52.

**Comment by flibitijibibo** [comment]
After the style fix it now makes a lot more sense... that was a really good call. And yeah, if existing data never used it, and the single-player data is okay, this should be safe enough.

**Pull request merged**